### PR TITLE
add arbitrum subgraphs

### PR DESCRIPTION
**Pull request by jmzwar**
Add subbies for arbitrum one and testnet

### CODE REVIEW

File: protocol/synthetix/subgraph/subgraph.arbitrum-sepolia.yaml [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+      address: "0x45f3822fc5F486F28F3F33B1d602995a0d4A3944"
+      startBlock: 21127604
+    mapping:
+      kind: ethereum/events
+      apiVersion: 0.0.6
+      language: wasm/assemblyscript
+      file: ./arbitrum-sepolia/index.ts
+      abis:
+        - name: CoreProxy
+          file: ./arbitrum-sepolia/deployments/system/CoreProxy.json
```
Comment (noisekit) [inline protocol/synthetix/subgraph/subgraph.arbitrum-sepolia.yaml:19]: arb sepolia is stil borked afaik, suggest removing it for the time being. Testing straight on the mainnet!

File: protocol/synthetix/subgraph/package.json [hunk 2]
```diff
@@ -16,6 +16,8 @@
     "deploy:optimism-goerli": "graph deploy snx-v3/optimism-goerli subgraph.optimism-goerli.yaml --output-dir ./build/optimism-goerli --product hosted-service",
     "alchemy:base-mainnet-andromeda": "graph deploy synthetix-base-mainnet-andromeda subgraph.base-mainnet-andromeda.yaml --output-dir ./build/base-mainnet-andromeda --version-label v1 --node https://subgraphs.alchemy.com/api/subgraphs/deploy --deploy-key $SATSUMA_KEY --ipfs https://ipfs.satsuma.xyz",
     "alchemy:base-sepolia-andromeda": "graph deploy synthetix-base-sepolia-andromeda subgraph.base-sepolia-andromeda.yaml --output-dir ./build/base-sepolia-andromeda --version-label v1 --node https://subgraphs.alchemy.com/api/subgraphs/deploy --deploy-key $SATSUMA_KEY --ipfs https://ipfs.satsuma.xyz",
+    "alchemy:arbitrum-one": "graph deploy synthetix-arbitrum-one subgraph.arbitrum-one.yaml --output-dir ./build/arbitrum-one --version-label v1 --node https://subgraphs.alchemy.com/api/subgraphs/deploy --deploy-key $SATSUMA_KEY --ipfs https://ipfs.satsuma.xyz",
```
Comment (noisekit) [inline protocol/synthetix/subgraph/package.json:19]: Lets keep the naming consistent. arbitrum-one -> arbitrum-mainnet please.
```suggestion
    "alchemy:arbitrum-mainnet": "graph deploy synthetix-arbitrum-mainnet subgraph.arbitrum-mainnet.yaml --output-dir ./build/arbitrum-mainnet --version-label v1 --node https://subgraphs.alchemy.com/api/subgraphs/deploy --deploy-key $SATSUMA_KEY --ipfs https://ipfs.satsuma.xyz",
```

Comment (jmzwar) [reply protocol/synthetix/subgraph/package.json:19]: Ok yeah I'm fine with that

Comment (jmzwar) [reply protocol/synthetix/subgraph/package.json]: Updated

**Review by noisekit — APPROVED** [review]

**Pull request merged**